frontend/src/components/worksheets/items/TableItem/BundleRow.js

```diff
@@ -238,7 +238,14 @@ class BundleRow extends Component {
 
          // Keyboard opening/closing
          if (this.props.focused) {
-             Mousetrap.bind(['enter'], (e) => this.setState((state) => ({ showDetail: !state.showDetail })), 'keydown');
+             Mousetrap.bind(
+                ['enter'], 
+                (e) => {
+                    e.preventDefault();
```

Comment:
Can you add a comment explaining why we do this? (so that selected links are not opened)